### PR TITLE
update tableland deps, fix chainId typing

### PR DESCRIPTION
**Pull request by joewagner**
@sanderpick I manually added what you have in your branch here.

## Overview 
Aside from bumping the tableland deps, there are a few little cleanups here. The only necessary one is doing proper type casting of request params before getting table data from the Validator.

## Details
I don't know how the Vercel deploy works, or if it works.  I would think by merging to `main` this will auto deploy, but we may want to double check things after merge.

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ app.use((req, res, next) => {
 
 app.get("/:chainId/:tableId", async (req, res, next) => {
   try {
-    const chainId = req.query.chainId || req.params.chainId;
```
Comment (joewagner) [inline api/index.js:41]: These are strings, and we now need `chainId` to be a number.

File: test/app.test.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { describe, test, afterEach, before } from "mocha";
 import { assert } from "sinon";
 import fetch from "node-fetch";
 
-describe("SVG Renderer", function () {
```
Comment (joewagner) [inline test/app.test.js:5]: There was two describes with the same text. This seemed confusing, but we can switch back if anyone has a strong preference.

File: test/setup.js [hunk 3]
```diff
@@ -13,8 +13,7 @@ if (!globalThis.fetch) {
 
 before(async function () {
   this.timeout(30000);
-  lt.start();
-  await lt.isReady();
+  await lt.start();
```
Comment (joewagner) [inline test/setup.js:16]: `await`ing `start` automatically waits till the network is ready now.

**Review by sanderpick — APPROVED** [review]
Great! Thanks for the assist. The only other thing I had in the other PR was README changes, but I can open another PR for that. It looked like it was written by an old version of chatgpt 😂

**Comment by joewagner** [comment]
> Great! Thanks for the assist. The only other thing I had in the other PR was README changes, but I can open another PR for that. It looked like it was written by an old version of chatgpt 😂

Ha!  
I missed the README update, but another PR sounds good to me.  
@sanderpick Do you have the ability to look at Vercel and see why this PR's deploy seems stuck?

**Comment by sanderpick** [comment]
for some reason the name of the deploy job changed. i fixed it in the branch protection rules. g2g now

**Comment by sanderpick** [comment]
I also checked the preview deploy. looks good

**Pull request merged**